libandroid-navigation/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,12 @@
     <string name="miles">mi</string>
     <string name="feet">ft</string>
     <string name="eta_format">%s ETA</string>
+
+    <!-- Time formatting -->
+    <string name="day">day</string>
```

Comment:
Should we use [quantity string (plurals)](https://developer.android.com/guide/topics/resources/string-resource#Plurals) instead of having multiple "day" strings?